contracts/ChargedParticles.sol

```diff
@@ -117,10 +128,29 @@ contract ChargedParticles is
     return address(_chargedSettings);
   }
 
+  function getManagersAddress() external view virtual override returns (address managersAddress) {
+    return address(_chargedManagers);
+  }
+
   function onERC721Received(address, address, uint256, bytes calldata) external virtual override returns (bytes4) {
     return IERC721ReceiverUpgradeable(0).onERC721Received.selector;
   }
 
+  function onERC1155Received(address, address, uint256, uint256, bytes calldata) external virtual override returns (bytes4) {
+    return IERC1155ReceiverUpgradeable(0).onERC1155Received.selector;
+  }
+
+  // Unimplemented
+  function onERC1155BatchReceived(address, address, uint256[] calldata, uint256[] calldata, bytes calldata) external virtual override returns (bytes4) {
+    return ""; // IERC1155ReceiverUpgradeable(0).onERC1155BatchReceived.selector;
+  }
+
+  function supportsInterface(bytes4 interfaceId) external view virtual override returns (bool) {
```

Comment:
was this just missing from V1? seems incredibly useful, confused as why it wasn't used before